### PR TITLE
docs: add globegitter to Angular collaborators

### PR DESCRIPTION
**Pull request by alexeagle**
fyi @globegitter

### CODE REVIEW

**Comment by alexeagle** [comment]
We'll need to get a signed NDA from you so we can invite you to meetings. @manughub will send one.

**Comment by alexeagle** [comment]
@manughub says the NDA is signed, thanks!

**Comment by manughub** [comment]
Sent another reminder for NDA acceptance. Setting PR back to blocked.

**Comment by Globegitter** [comment]
Ah sorry, I just saw this now, still taking me some time to get used to the way github ui shows me PRs I have been mentioned in. Will sort this out asap.

**Comment by jenniferfell** [comment]
@alexeagle : Is there a reason this is master only, and not master & patch?

**Comment by Globegitter** [comment]
I signed the NDA now.

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -643,5 +643,11 @@
     "website": "https://medium.com/@zizzamia",
     "bio": "Leonardo is a Senior Software Engineer at Coinbase. He is deeply passionate about web performance and most recently developed Perfume.js to help companies prioritize roadmaps and business, through performance analytics. From 2018, co-organizer of the Angular San Francisco Meetup group and NGRome Conference.",
     "group": "GDE"
+  },
+  "globegitter": {
+    "name": "Markus Padourek",
+    "group": "Collaborator",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: After #29553, this now needs to be changed to: `groups: ["Collaborators"]`

**Review by gkalpak — DISMISSED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (alexeagle) [reply aio/content/marketing/contributors.json]: done

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**